XMLHttpRequest/resources/auth8/corsenabled-no-authorize.py

```diff
@@ -0,0 +1,20 @@
+import imp
+import os
+
+def main(request, response):
+    response.headers.set('Access-Control-Allow-Origin', request.headers.get("origin"));
+    response.headers.set('Access-Control-Allow-Credentials', 'true');
+    response.headers.set('Access-Control-Allow-Methods', 'GET');
+    response.headers.set('Access-Control-Allow-Headers', 'x-user, x-pass');
+    response.headers.set('Access-Control-Expose-Headers', 'x-challenge, xhr-user, ses-user');
```

Comment:
I'm tempted to request that we reuse the previous corsenabled.py and add use a query parameter to dictate whether the `authorization` header is exposed instead. I think that would make it easier to follow this test.